plugins/modules/vyos_lldp_interfaces.py

```diff
@@ -158,18 +167,19 @@
             longitude: 222.267255W
             latitude: 33.524449N
     state: merged
-
 #
 #
 # -------------------------
 # Module Execution Result
 # -------------------------
 #
-# before": []
+# before": {}
 #
 #    "commands": [
 #        "set service lldp interface eth1 location civic-based country-code 'US'",
 #        "set service lldp interface eth1 location civic-based ca-type 0 ca-value 'ENGLISH'",
+#        "set service lldp interface eth1 location civic-based country-code 'US'",
+#        "set service lldp interface eth1 location civic-based ca-type 0 ca-value 'ENGLISH'",
```

Comment:
Invalid path for 1.3.8
```
vyos@client-11# set service lldp interface eth1 location civic-based 

  Configuration path: service lldp interface eth1 location [civic-based] is not valid

```

Reply:
lldp hasn't been updated to 1.3.8 from 1.2 yet. That will come when we update the code for 1.3+